src/libs/error.ts

```diff
@@ -1,12 +1,14 @@
 import { AnyPromiseFn, GenericPromiseFn, safeCurry } from '@terra-ui-packages/core-utils';
 import _ from 'lodash/fp';
+import { sessionTimedOutErrorMessage } from 'src/auth/auth-errors';
 import { notify } from 'src/libs/notifications';
 
 export const reportError = async (title, obj) => {
   console.error(title, obj); // helpful when the notify component fails to render
   // Do not show an error notification when a session times out.
   // Notification for this case is handled elsewhere.
-  if (obj instanceof Error && obj.message === 'Session timed out') {
+  // For some reason making using sessionTimedOutErrorMessage instead of the raw string breaks tests
```

Comment:
```suggestion
```

This comment is no longer true since the import cycle has been resolved by moving `sessionTimedOutErrorMessage` to another module.